From 41a3b6659508f9c251ba741e757bc8cb6b617776 Mon Sep 17 00:00:00 2001
Message-Id: <41a3b6659508f9c251ba741e757bc8cb6b617776.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Tue, 28 Oct 2014 17:38:14 +0000
Subject: [PATCH 05/10] REVERTME [VPG]: drm/i915: Remove unused variables in
 MMIO flip

Fix the fumble in:
commit a93fc67c0aba47bdee598e008ad2811bf10c4f95
Author: Vidya Srinivas <vidya.srinivas@intel.com>
Date:   Fri Sep 26 10:14:52 2014 +0530

    FOR_UPSTREAM [VPG]: drm/i915: Fix in the MMIO flip for VLV

For: GMINL-1778
Change-Id: I2ea4456e116c2aa3d49fa367553f4fbb8715c291
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 124d927..b47bc7b 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -10017,13 +10017,8 @@ static void intel_do_mmio_flip(struct intel_crtc *intel_crtc)
 {
 	struct drm_device *dev = intel_crtc->base.dev;
 	struct drm_i915_private *dev_priv = dev->dev_private;
-	struct intel_framebuffer *intel_fb =
-		to_intel_framebuffer(intel_crtc->base.primary->fb);
-	struct drm_i915_gem_object *obj = intel_fb->obj;
 	struct drm_crtc *crtc =
 		dev_priv->pipe_to_crtc_mapping[intel_crtc->pipe];
-	u32 dspcntr;
-	u32 reg;
 
 	intel_mark_page_flip_active(intel_crtc);
 	dev_priv->display.update_primary_plane(crtc,
-- 
1.7.9.5

